modules/asg/scripts/lambda.py

```diff
@@ -180,7 +180,7 @@ def inspect_ec2_instance(self, instance_id: str) -> tuple:
         return instance_info.get('Placement').get('AvailabilityZone') if 'Placement' in instance_info else None, \
             instance_info.get('SubnetId'), instance_info.get('NetworkInterfaces')
 
-    def create_network_interface(self, instance_id: str, subnet_id: str, sg_id: int) -> str:
+    def create_network_interface(self, instance_id: str, subnet_id: str, sg_id: list) -> str:
```

Comment:
```suggestion
    def create_network_interface(self, instance_id: str, subnet_id: str, sg_ids: list) -> str:
```

Reply:
Besides that I propose to change:
- in line 189:  ```:param sg_ids: Security group ids``` instead of ```:param sg_id: Security group id```
- in line 193: ```sg_ids={sg_ids}``` instead of ```sg_id={sg_id}```

Reply:
Thanks a lot @sebastianczech, I agree it makes good sense to change the parameter name to plural now. Done!